### ISSUE TITLE
Not working after update

### ISSUE THREAD

**Issue by djerveren** [antiwinter/wowa#34]
Just updated and wowa no longer works. `wowa` throws the following error no matter what I do:
 ```
$ sudo npm install -g wowa
/usr/local/bin/wowa -> /usr/local/lib/node_modules/wowa/index.js
+ wowa@1.3.10
added 1 package and updated 13 packages in 15.297s

$ wowa update
internal/modules/cjs/loader.js:1174
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /usr/local/lib/node_modules/wowa/node_modules/is-promise/index.js
require() of ES modules is not supported.
require() of /usr/local/lib/node_modules/wowa/node_modules/is-promise/index.js from /usr/local/lib/node_modules/wowa/node_modules/listr/lib/task.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /usr/local/lib/node_modules/wowa/node_modules/is-promise/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1174:13)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/wowa/node_modules/listr/lib/task.js:2:19)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14) {
  code: 'ERR_REQUIRE_ESM'
}

$ lsb_release -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	Fedora
Description:	Fedora release 31 (Thirty One)
Release:	31
Codename:	ThirtyOne
```

**Comment by antiwinter** [comment]
Which node version are you using? `node -v`

**Comment by antiwinter** [comment]
This seems to be an [is-promise](https://github.com/then/is-promise) bug on version 2.2.0 which has been fixed on v2.2.1.  This is their issue https://github.com/then/is-promise/issues/14, https://github.com/then/is-promise/issues/20.

You can try installing **wowa** again by using `npm install -g wowa` and see if the problem still exists.

**Comment by djerveren** [comment]
```
node -v
v12.16.1
```

Works after upgrading wowa again. Thanks for the speedy response! :)
